packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -68,7 +69,10 @@ const routes = (
       element={<TechRadarPage width={1500} height={800} />}
     />
     <Route path="/catalog-import" element={<CatalogImportPage />} />
-    <Route path="/search" element={<SearchPage />} />
+      <Route path="/search" element={<SearchPage />}>
+      {searchPage}
+    </Route>
+    ;
```

Comment:
Formatting here is a bit off, which is failing CI

Reply:
These changes are already mentioned in the CHANGELOG.md 

https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md#0330 
https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md#0325

but files are not updated in `create-app` 

Fixed the white spaces in https://github.com/backstage/backstage/pull/6931/files#diff-c6453d0dd45f236e992074aae7ead2217f5fc579a4a3420f2ed66af9a02c7df2R72

Reply:
@freben In the change log at https://github.com/backstage/backstage/blob/master/packages/create-app/CHANGELOG.md#0325 - There is a semi column. Please let me know if I have to still fix?

Reply:
yeah for sure, otherwise there will be an actual semicolon in the page

Reply:
To be clear the semicolon in the changelog is incorrect and we should remove it from there too